src/components/SearchToolbar/SearchResults.js

```diff
@@ -58,6 +70,22 @@ const StyledSearchResults = styled(SearchResults)`
     }
   }
 
+  li.no-result {
+    text-align: center;
+    font-size: 16px;
+    overflow: hidden;
+    padding: 20px;
+  }
+
+  li.error-result {
+    text-align: center;
+    font-size: 16px;
+    overflow: hidden;
+    padding: 20px;
+    font-weight: 400;
+    background-color: #ffbab2;
```

Comment:
Mini cosmetics wish: Could you use something from `colors.js` or put this color into `colors.js`?